application/world/services/auto_bible_generator.py

```
@@ -679,15 +539,13 @@ async def _save_worldbuilding(self, novel_id: str, worldbuilding_data: Dict[str,
                     culture=worldbuilding_data.get("culture"),
                     daily_life=worldbuilding_data.get("daily_life")
                 )
-                print(f"[DEBUG] Worldbuilding saved to Worldbuilding table", file=sys.stderr, flush=True)
-                logger.info(f"Worldbuilding saved for {novel_id}")
+                logger.info(f"Worldbuilding saved to Worldbuilding table")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)